airbyte-webapp/src/hooks/services/BulkEdit/BulkEditService.tsx

```diff
@@ -57,6 +60,9 @@ export const BulkEditServiceProvider: React.FC<
   };
 
   const isActive = selectedBatchNodes.size > 0;
+  useEffect(() => {
```

Comment:
This effect should be moved above line 43 so that it's below the existing hooks and above non-hooks.